CMakeLists.txt

```diff
@@ -14,17 +14,25 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
-cmake_minimum_required(VERSION 3.5.1)
+cmake_minimum_required(VERSION 3.9)
 project("RHVoice")
 
-set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${CMAKE_SOURCE_DIR}/cmake")
+set(customCMakeModulesDir "${CMAKE_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${customCMakeModulesDir}" "${customCMakeModulesDir}/thirdparty/MinGWSAPI5SDK" "${customCMakeModulesDir}/thirdparty/CPackRegistry")
```

Comment:
how about hiding this under if(mingw) or if(windows) too?
I bet it is not needed on the non-windows OSes

Reply:
Thanks, probably should do that.